packages/esm-patient-labs-app/src/lab-orders/add-lab-order/lab-order-form.component.tsx

```diff
@@ -171,21 +195,36 @@ export function LabOrderForm({ initialOrder, closeWorkspace }: LabOrderFormProps
                   name="instructions"
                   control={control}
                   render={({ field: { onChange, onBlur, value } }) => (
-                    <TextInput
+                    <TextArea
+                      counterMode="character"
+                      enableCounter={true}
```

Comment:
Nit: truthy values for boolean props can be safely omitted. `enableCounter` === `enableCounter={true}` with the benefit of being less verbose.

Reply:
Also, it looks like the `counterMode` prop for TextArea is only supported on the latest versions of Carbon:

![CleanShot 2024-02-09 at 12  57 14@2x](https://github.com/openmrs/openmrs-esm-patient-chart/assets/8509731/ff45c9c0-7e9e-4858-9de8-42b0bbb0a924)

Does the input work just as well without it? We should likely omit it until we upgrade to a newer version.

Reply:
Yeah, It works without it, I have omitted it
cc @denniskigen